sdcm/results_analyze.py

```diff
@@ -175,6 +175,136 @@ def check_regression(self):
         return NotImplementedError("check_regression should be implemented!")
 
 
+class SpecificResultsAnalyzer(BaseResultsAnalyzer):
```

Comment:
need a better name, I don't understand what it does

Reply:
changed to: "SpecificStatsPerformanceAnalyzer". 
it means that the user sends his own stats to get ES results for.